src/spyglass/lfp/analysis/v1/lfp_band.py

```diff
@@ -414,57 +540,51 @@ def compute_analytic_signal(self, electrode_list: list[int], **kwargs):
             index=pd.Index(filtered_band.timestamps, name="time"),
             columns=[f"electrode {e}" for e in electrode_list],
         )
+
         return analytic_signal_df
 
     def compute_signal_phase(
-        self, electrode_list: list[int] = None, **kwargs
+        self, electrode_list: Optional[list[int]] = None
```

Comment:
Intentionally dropping `**kwargs`? Potential consequence of erroring when called by some external func that passes extra unused params

Reply:
I couldn't find an instance where this was used so I dropped it.